lale/lib/sklearn/dummy_regressor.py

```diff
@@ -0,0 +1,89 @@
+# Copyright 2019 IBM Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import sklearn
+import sklearn.dummy
+
+import lale.docstrings
+import lale.operators
+from lale.schemas import AnyOf, Enum, Float, Null
+
+
+
+_hyperparams_schema = {
+    "allOf": [
+        {
+            "description": "This first object lists all constructor arguments with their types, but omits constraints for conditional hyperparameters.",
```

Comment:
please add a link to the underlying sklearn docs here (an example doing this is https://github.com/IBM/lale/blob/master/lale/lib/sklearn/logistic_regression.py#L397)